pkgs/development/python-modules/apache-beam/default.nix

```diff
@@ -62,16 +63,31 @@
 
 buildPythonPackage rec {
   pname = "apache-beam";
-  version = "2.66.0";
+  version = "2.67.0";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "apache";
     repo = "beam";
     tag = "v${version}";
-    hash = "sha256-nRofy9pvhO5SUvkIk73ViFm1gPWxEhj1rAUeCVYIpYs=";
+    hash = "sha256-gHlbmPNtSKjYO4YPPtWnjul977APsHwe/2cq2UG4wuk=";
   };
 
+  sourceRoot = "${src.name}/sdks/python";
+
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace-fail "distlib==0.3.7" "distlib" \
+      --replace-fail "yapf==0.43.0" "yapf" \
+      --replace-fail "grpcio-tools==1.62.1" "grpcio-tools" \
+      --replace-fail "grpcio-tools==1.71.0" "grpcio-tools" \
+      --replace-fail "mypy-protobuf==3.5.0" "mypy-protobuf" \
```

Comment:
Does
```suggestion
      --replace-fail "==" ">=" \
```
work?

Reply:
Had to handle `numpy`'s `<2.3.0`, but it worked for the rest.